### DISCUSSION TITLE
Website Design

### DISCUSSION THREAD

**Discussion by blaccsmith** [blaccsmith/website#23] [Ideas]
We're currently in discussion of the finalized website design 🎨

## Design 1
![homepage-1](https://user-images.githubusercontent.com/88869469/130340850-9b8c9e51-64a0-4d24-868b-170a7fb34495.png)

## Design 2
![homepage-2](https://user-images.githubusercontent.com/88869469/130340851-85d6e96c-4c7c-4a3d-b924-166408919aee.png)

## Design 3
![homepage-3](https://user-images.githubusercontent.com/88869469/130340852-f49d5530-d886-4387-8e34-8dde72e89c06.png)

We would love to hear which is your favorite and any new features or styles that you'd want to see down below !

**Comment by Genyus** [comment]
Visually, no.3 definitely stands out to me the most. But I love the idea of the “city cloud” on no.2, maybe just not as a hero element.

Testimonials styling looks great, but not entirely sure on what the “24/7” highlighting is supposed to represent?

**Reply by gt-codes** [reply]
The 24/7 is the hover state. Those are meant to be tiles and you see the tile "pop out" when you hover over it. Meant to be a nice little Easter egg 😅

**Reply by Genyus** [reply]
Ah cool, I gotchu! That works 👍🏾
